### PR TITLE
Enhancement/ConfirmSelectionButton

### PR DESCRIPTION
**Pull request by m4rtint**
Screen Shot 2020-09-30 at 2 20 20 AM

### CODE REVIEW

**Comment by m4rtint** [comment]
[Add spritesheet of confirm keys into Player Selection (0.5)](https://trello.com/c/9a0WRg2V/903-add-spritesheet-of-confirm-keys-into-player-selection-05)

File: Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/ConfirmViewController.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+﻿using InControl;
```
Comment (Gursaanj) [inline Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/ConfirmViewController.cs:1]: Missing Header

**Review by Gursaanj — COMMENTED** [review]
Oh makes sense!!

File: Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/ConfirmViewController.cs (hunk 1 above)
Comment (m4rtint) [reply Assets/_PartyParkade/Script/UI/MainMenu/PlayerSelection/PlayerSelectionVC/ConfirmViewController.cs]: Updated

**Review by Gursaanj — APPROVED** [review]

**Review by Gursaanj — CHANGES_REQUESTED** [review]
image
Need to unsubscribe from these methods

**Comment by m4rtint** [comment]
> image
> Need to unsubscribe from these methods

replied on slack.
I null the viewModel, it's fine

**Review by Gursaanj — APPROVED** [review]

**Review by wbeltran777 — APPROVED** [review]
Looks all good

**Pull request merged**